### PR TITLE
Add 'ImportSoftEdgeRecord' job & test.

### PR DESCRIPTION
**Pull request by DFurnes**
### What's this PR do?

This pull request migrates the [`ImportSoftEdgeRecord`](https://github.com/DoSomething/chompy/blob/main/app/Jobs/SoftEdge/ImportSoftEdgeRecord.php) job from Chompy into Northstar.

### How should this be reviewed?

I've written a simple test case to validate this job works as expected.

### Any background context you want to provide?

I don't believe we're actively importing these, but this isn't functionality we want to lose. We'll follow up with the API endpoints for these third-party integrations after this, so that it's easy to re-enable when we renew these partnerships.

### Relevant tickets

References [Pivotal #177733342](https://www.pivotaltracker.com/story/show/177733342).

### Checklist

- [ ] This PR has been added to the relevant Pivotal card.
- [ ] Documentation added for new features/changed endpoints.
- [ ] Added appropriate feature/unit tests.
- [ ] If new attributes were added to users, there is a corresponding PR to surface these in Aurora.
- [ ] If new attributes were added to users, then the data team already knows about these changes.

### CODE REVIEW

**Review by mendelB — APPROVED** [review]
💥

File: app/Jobs/Imports/ImportSoftEdgeRecord.php [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+class ImportSoftEdgeRecord implements ShouldQueue
+{
+    use Dispatchable, InteractsWithQueue, Queueable;
+
+    /**
+     * The call parameters sent from SoftEdge.
+     *
+     * @var array
+     */
+    protected array $parameters;
```
Comment (weerd) [inline app/Jobs/Imports/ImportSoftEdgeRecord.php:25]: Is `array` supposed to be after the `protected` keyword? Maybe this is just a newer PHP thing?

Comment (DFurnes) [reply app/Jobs/Imports/ImportSoftEdgeRecord.php:25]: Yep, this just indicates the [type of the property](https://www.php.net/manual/en/language.oop5.properties.php#language.oop5.properties.typed-properties) in a way that the compiler can enforce.

Comment (weerd) [reply app/Jobs/Imports/ImportSoftEdgeRecord.php:25]: Ah nice! I only noticed it cause VSCode has it underlined in red! Thanks for the tip!

**Pull request merged**